src/data/skill-desc.csv

```diff
@@ -25,6 +25,7 @@
 "レース後半で\n1番をキープし続ける力を発揮する","<size=22>Display your power in keeping the lead position during the latter half of the race</size>"
 "最終コーナーで好位置にいると\n弾みを付けて加速力が上がる","<size=22>Be in a good position at the last corner to gain momentum and increase your acceleration</size>"
 "最終直線で後ろの方から追い抜くと\n末脚を使って速度が上がる","<size=22>Increase speed when passing in the back in Final Leg straight</size>"
+"最終直線で後ろの方から外を回って\n追い抜く時、速度を上げる","<size=22>On last straight, gain speed when passing from the outside back</size>"
```

Comment:
Add a "the" here, i.e. "On the left straight"

Reply:
yeah im cutting them to prevent the text from being too long

Reply:
Does it really not fit? Some other texts are longer, so adding a "the" there shouldn't be a big problem. Though if it is then yea ofc, leave it out.

Reply:
well it is as you said wouldnt be a problem, but then i again that means i need to add "the" for the other similar cases lol

Reply:
Guess it could stay like that then, and be included in a consistency PR if anyone wants to make that.